cmd/cloud_run.go

```diff
@@ -68,6 +93,33 @@ against the Grafana k6 cloud. Use the "k6 cloud login" command to authenticate.`
 	return cloudRunCmd
 }
 
+func (c *cmdCloudRun) flagSet() *pflag.FlagSet {
+	flags := pflag.NewFlagSet("", pflag.ContinueOnError)
+	flags.SortFlags = false
+
+	// FIXME @oleiade: we should make a passe over the flags defined there, and define our own set with the ones needed
+	// for local-execution.
+	flags.AddFlagSet(optionFlagSet())
+
+	// FIXME: this is brought directly from the `k6 run` command, and we might want to use a limited set of
+	// those flags here?
+	flags.AddFlagSet(runtimeOptionFlagSet(false))
+
+	// TODO: Figure out a better way to handle the CLI flags
+	flags.BoolVar(&c.exitOnRunning, "exit-on-running", c.exitOnRunning,
+		"exits when test reaches the running status")
```

Comment:
I think this one can be simply removed, as I think it doesn't make sense for `--local-execution`.

Reply:
Great idea, let's look into it 👍🏻